.github/workflows/coverage-check.yml

```diff
@@ -0,0 +1,51 @@
+name: Check Test Coverage
```

Comment:
According to this line it should be visible as "Check Test Coverage" in your actions but I don't see it. 

https://github.com/pwilson77/ubiquity-dollar/actions

Where can I see proof that this is working?

Reply:
@pavlovcik , I thought it was to be visible in the ubiquity dollar repository actions itself
https://github.com/ubiquity/ubiquity-dollar/actions/workflows/coverage-check.yml